example/facebook_places.js

```diff
@@ -72,7 +72,7 @@ exports.window = function(value) {
 
 	function findPlacesBySearchTerm(value) {
 	    fb.fetchNearbyPlacesForSearchTearm({
-	        searchTerm: value,
+	        searchTearm: value,
```

Comment:
No need to touch any of the places related Facebook API's. Facebook removed all places related features recently, see #115

Reply:
Just fixed typo mistakes, will update sample app once places related features will be removed from module OR once PR #115 will be merged.